webapi/src/main/scala/org/knora/webapi/responders/v1/GroupsResponderV1.scala

```diff
@@ -0,0 +1,348 @@
+            group_info = groupInfo,
+            userdata = userProfile match {
+                case Some(profile) => Some(profile.userData)
+                case None => None
+            }
+        )
+
+        Future(groupInfoResponse)
+    }
+
+    private def getGroupInfoForProjectAdminGroup(projectIRI: IRI, infoType: GroupInfoType, userProfile: Option[UserProfileV1]): Future[GroupInfoResponseV1] = {
+
+        val groupInfo = GroupInfoV1(
+            id = "-",
+            name = "ProjectAdmin",
+            description = Some("Default Project Admin Group"),
+            belongsToProject = projectIRI,
+            isActiveGroup = true,
+            hasSelfJoinEnabled = false,
+            hasPermissions = Vector.empty[GroupPermissionV1]
+        )
+
+        val groupInfoResponse = GroupInfoResponseV1(
+            group_info = groupInfo,
+            userdata = userProfile match {
+                case Some(profile) => Some(profile.userData)
+                case None => None
+            }
+        )
+
+        Future(groupInfoResponse)
+    }
+
+    private def getGroupInfoByNameFromTriplestore(projectIRI: IRI, groupName: String, infoType: GroupInfoType, userProfile: Option[UserProfileV1]): Future[GroupInfoResponseV1] = {
+        for {
+            sparqlQuery <- Future(queries.sparql.v1.txt.getGroupByName(
+                triplestore = settings.triplestoreType,
+                name = SparqlUtil.any2SparqlLiteral(groupName),
+                projectIri = projectIRI
+            ).toString())
+            _ = log.debug(s"getGroupInfoByNameGetRequest - getGroupByName: $sparqlQuery")
+            groupResponse <- (storeManager ? SparqlSelectRequest(sparqlQuery)).mapTo[SparqlSelectResponse]
+
+            _ = log.debug(s"group response: ${groupResponse.toString}")
+
+            // check group response and get group Iri
+            groupIri: IRI = if (groupResponse.results.bindings.nonEmpty) {
+                groupResponse.results.bindings.head.rowMap("s")
+            } else {
+                throw NotFoundException(s"For the given group name '$groupName' no information was found")
+            }
+
+            // get group info
+            groupInfo = createGroupInfoV1FromGroupResponse(
+                groupResponse = groupResponse.results.bindings,
+                groupIri = groupIri,
+                infoType = infoType,
+                userProfile
+            )
+
+            groupInfoResponse = GroupInfoResponseV1(
+                group_info = groupInfo,
+                userdata = userProfile match {
+                    case Some(profile) => Some(profile.userData)
+                    case None => None
+                }
+            )
+        } yield groupInfoResponse
+    }
+
+    private def createGroupV1(newGroupInfo: NewGroupInfoV1, userProfile: UserProfileV1, apiRequestID: UUID): Future[GroupOperationResponseV1] = {
+
+        for {
+            a <- Future("")
```

Comment:
Please don't use dummy values, they're confusing and unnecessary. Instead, you can do this:

```
_ <- {
    if (newGroupInfo.name.isEmpty) throw BadRequestException("Group name cannot be empty")
    if (newGroupInfo.belongsToProject.isEmpty) throw BadRequestException("Project IRI cannot be empty")
}